runtime/oti/j9port.h

```diff
@@ -177,7 +177,15 @@
 #define J9SH_SEMAPHORE_ID "_semaphore_"
 
 #define J9SH_DIRPERM_ABSENT ((uintptr_t)-2)
+#define J9SH_DIRPERM_ABSENT_GROUPACCESS ((uintptr_t)-3)
+#if defined(OPENJ9_BUILD)
+#define J9SH_DIRPERM (0700)
+#define J9SH_DIRPERM_GROUPACCESS (0750)
+#else
 #define J9SH_DIRPERM (0777)
```

Comment:
Like the changeDirectoryPermission() comment, I'm thinking 777 should only be used when using a default /tmp/javasharedresources directory, anything else can use 700. Similarly for group access.